src/HTMLElement/elements/HTMLAnchorElement.js

```diff
@@ -0,0 +1,169 @@
+import HTMLElement from '../../HTMLElement';
+import Url from '../../utils/Url';
+
+/**
+ * The HTMLAnchorElement interface represents hyperlink elements and provides special properties and methods
+ * (beyond those of the regular HTMLElement object interface they also have available to them by inheritance)
+ * for manipulating the layout and presentation of such elements.
+ *
+ * @see https://developer.mozilla.org/en/docs/Web/API/HTMLAnchorElement
+ * @class HTMLAnchorElement
+ */
+export default class HTMLAnchorElement extends HTMLElement {
+    constructor() {
+        super();
+        this._href = new Url();
+    }
+    /**
+     * @inheritDoc
+     */
+    setAttribute(key, value) {
+        if (key === 'href') {
+            this.href = value;
+        } else {
+            super.setAttribute(key, value);
+        }
+    }
+
+    getAttribute(key) {
+        if (key === 'href') {
```

Comment:
Ok, seems first half is right. So we need to updated setter only. Ok i will fix this.